client/client.py

```diff
@@ -0,0 +1,103 @@
+"""
+Base class which contains the basic methods to interaction with the write api of NodeBB.
+"""
+import json
+
+import requests
+import urlparse
+from django.conf import settings as django_settings
+
+
+class Client(object):
+    """
+    Client Class responsible to make connection with NodeBB and perform all calls.
+    """
+    def __init__(self):
+        self._configure()
+
+    def _configure(self):
+        self.endpoint = 'http://172.16.16.24:4568'
+        self.admin_uid = django_settings.OPENEDX_NODEBB_DISCUSSION['NODEBB_ADMIN_UID']
```

Comment:
The admin_uid is always 1. So, I believe it can be defined as a constant in the client.

Reply:
Done